addons/website_sale/views/templates.xml

```diff
@@ -2910,7 +2727,7 @@
                                 </div>
                             </div>
                         </div>
-                        <div t-attf-class="oe_cart col-12 col-lg-7">
+                        <div t-attf-class="oe_clear_stucture oe_cart col-12 col-lg-7">
```

Comment:
What does this class do? I'd rather not adding a random class to the e-commerce checkout layout if not needed (this affects cart, address, checkout, payment, confirmation, etc.)
```suggestion
                        <div t-attf-class="oe_clear_structure oe_cart col-12 col-lg-7">
```

Reply:
It contains same style as `oe_cart` only needed to move in portal
(Also kept `oe_cart` also as it was used in selectors)

Reply:
Thanks for the clarification! 

Please either fully replace `oe_cart` by `oe_clear_structure` or just don't add `oe_clear_structure` then. Having two classes that does the same thing is useless. 

Also, if you choose to remove `oe_cart`, please replace selectors with a name or an id. It's best to use classes solely for styling.